code/game/gamemodes/clock_cult/clock_unsorted.dm

```diff
@@ -140,8 +140,8 @@
 			unconverted_ai_exists = TRUE
 	. = list(SCRIPTURE_DRIVER = TRUE, SCRIPTURE_SCRIPT = FALSE, SCRIPTURE_APPLICATION = FALSE, SCRIPTURE_REVENANT = FALSE, SCRIPTURE_JUDGEMENT = FALSE)
 	//Drivers: always unlocked
-	.[SCRIPTURE_SCRIPT] = (servants >= 5 && clockwork_caches)
```

Comment:
I didn't expect byond to produce a 'true' value for each time this was true, or in other words it'd have a value of 5 at five caches and 5+ servants.